tools/apigee-sackmesser/cmd/export/export.sh

```diff
@@ -80,15 +83,17 @@ sackmesser list "organizations/$organization/environments" | jq -r -c '.[]|.' |
 
     mkdir -p "$export_folder"/environments/"$env"/targetservers
     sackmesser list "organizations/$organization/environments/$env/targetservers" | jq -r -c '.[]|.' | while read -r targetserver; do
-        sackmesser list "organizations/$organization/environments/$env/targetservers/$targetserver" | jq '.' > "$export_folder"/environments/"$env"/targetservers/"$targetserver".json
+        sackmesser list "organizations/$organization/environments/$env/targetservers/${targetserver/ /%20}" | jq '.' > "$export_folder"/environments/"$env"/targetservers/"${targetserver/ /-}".json
```

Comment:
is space the only character that needs encoding? we could be safer by running a urlencode function on the whole variable

Reply:
ack, using jq to encode the uri component correctly.